### ISSUE TITLE
Notifikace u batůžku / košíku

### ISSUE THREAD

**Issue by kubaran11** [Stredni-skola-managementu-a-grafiky/UP3AI#43]
Vytvořte JS kód, který při použití třídy "active" zobrazí červenou notifikaci u batůžku a když tam tato třída není červená notifikace se nebude zobrazovat.

**Comment by Onderko** [comment]
``var noti = document.querySelector('h1');

/* <h1>
<span>
<img src="./images/bag.png" height="24px" alt="" />
        
  </h1> */

	var select = document.querySelector('.select');
	var button = document.getElementsByTagName('button');
    //  <button>Do košíku</button>
	for(var but of button){
		but.addEventListener('click', (e)=>{
			var add = Number(noti.getAttribute('data-count') || 0);
			noti.setAttribute('data-count', add +1);
			noti.classList.add('zero')

			if (clone) {
				noti.onclick = ()=>{
					select.classList.toggle('display');
				}	
			}
		})
	}

**Comment by kubaran11** [comment]
@Onderko querySelector na "h1" není asi úplně ideální filtr, h1 nadpisů tam může být víc a pak ti to vezme ten víc nahoře, asi bych to více specifikoval.

**Comment by Onderko** [comment]
@kubaran11 dobrá změním to dneska popřípadě tomu hodím  pozadí at to má trošku vzhled, ale nebylo by přece jednoduší udělat potom košík přes php kde si bude držet uživatel věci pod svým id v košíku třeba přeš count_item();?

**Comment by kubaran11** [comment]
> @kubaran11 dobrá změním to dneska popřípadě tomu hodím pozadí at to má trošku vzhled, ale nebylo by přece jednoduší udělat potom košík přes php kde si bude držet uživatel věci pod svým id v košíku třeba přeš count_item();?

Jo, tak to určitě bude. Je na to připravený i databázový návrh #1 .
